branding/src/main/scala/com/normation/rudder/rest/BrandingApiSchema.scala

```diff
@@ -64,13 +63,15 @@ object BrandingApiEndpoints extends ApiModuleProvider[BrandingApiSchema] {
     val description    = "Update branding plugin configuration"
     val (action, path) = POST / "branding"
     val dataContainer: Option[String] = None
+    override val name: String         = "updateBRandingConf" // the initial version of the plugin used that name
```

Comment:
Indeed but i think we can fix it. It's an api from plugin that is not used much, especially from api and we better not keep it 

That's good rule to keep it in most cases but this one is really not dangerous ... and i can live with that typo if we decide to keep it

Reply:
Okay, I did not take the risk because I didn't know the use case, but I'm changing it then :rocket: 
 
Also, there is the endpoint `ReloadBrandingConf` that currently has the same name as `GetBrandingConf`, I guess I can change it on the same basis to have a name _reloadBrandingConf_

Reply:
Yes totally !

Reply:
And as I said, not changing the value is in general cases the good move indeed !!